package.json

```diff
@@ -71,6 +72,13 @@
     "setupFilesAfterEnv": [
       "./tests/setup/setup.js"
     ],
+    "moduleFileExtensions": [
+      "js"
+    ],
+    "moduleDirectories": [
+      "node_modules",
+      "<rootDir>"
+    ],
```

Comment:
this lets you do something like 
`import FilterNodeFactory from 'src/core/filters/filternodefactory';`
instead of
`import FilterNodeFactory from '../../../../src/core/filters/filternodefactory';`

Reply:
the default is just ["node_modules"]

Reply:
Does this also work for the source code as well? That could be nice.

Reply:
Ah this is only for jest tests I think we'd have to change how our grunting or gulping for that